tensorflow/lite/tools/pip_package/build_pip_package.sh

```diff
@@ -23,7 +23,11 @@ export TENSORFLOW_DIR="${SCRIPT_DIR}/../../../.."
 TENSORFLOW_LITE_DIR="${TENSORFLOW_DIR}/tensorflow/lite"
 TENSORFLOW_VERSION=$(grep "_VERSION = " "${TENSORFLOW_DIR}/tensorflow/tools/pip_package/setup.py" | cut -d= -f2 | sed "s/[ '-]//g")
 export PACKAGE_VERSION="${TENSORFLOW_VERSION}${VERSION_SUFFIX}"
-BUILD_DIR="/tmp/tflite_pip/${PYTHON}"
+if [[ "${TENSORFLOW_CROSS_COMPILATION}" == "true" ]]; then
```

Comment:
how to set TENSORFLOW_CROSS_COMPILATION?

Reply:
Hi,
TENSORFLOW_CROSS_COMPILATION is a new variable introduce to not break the current way of building Tensorflow pip package.
It should be set by the compilation environment to warn the .sh script that the build directory should change to have a local build directory.

I'd rather like to not have this environment variable but in this case, the BUILD_DIR should be a local build directory for every one meaning BUILD_DIR="$'{SCRIPT_DIR}/gen/tflite_pip" 

What would you prefer?

Reply:
What's the issue of using "/tmp/tflite_pip/${PYTHON}" for cross build?

BTW, we also need to update tensorflow/lite/tools/pip_package/Makefile to change it.

Reply:
In some cross compilation environment (like Yocto OpenEmbedded), you don't have access the the root directory of your host computer to build. It is impossible to use /tmp/.

Yes your right about the Makefile if you decide to mode the build directory into a local directory.

Do you want me to update my patch taking into account only a local directory for the build?

Reply:
I see. Yes, let's use local directory to build which is aligned with lite/tools/make/Makefile behavior. Could you apply it?

Reply:
yes for sure, new patch will be available soon

Reply:
New patch uploaded